getting-started/templates/GCP/gcp-supplemental-values.yaml

```diff
@@ -0,0 +1,385 @@
+          cpu: "1"
+
+  ## Extra volumes that can be used in sidecars
+  extraVolumes:
+    # This volume is required for the proxy to authenticate with cloudSQL using a service account key file. 
+    - name: <token-volume-name> # <ATTENTION> - Enter the volume name where the token is to be mounted
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: <audience-name> # <ATTENTION> - Enter the audience name for the projected service account token
+              expirationSeconds: 3600
+              path: token
+    # This volume is required for the proxy to authenticate with cloudSQL when using Workload Identity Federation.
+    - name: <config-volume-name> # <ATTENTION> - Enter the volume name where config.json is to be mounted
+      secret:
+        secretName: <secret-name> # <ATTENTION> - Enter the secret name where config.json is added.
+
+  connectionInfo:
+    ## @param database.connectionInfo.host PostgreSQL hostname. Since the cloud-sql-auth-proxy is used, this value should be localhost.
+    ##
+    host: "localhost"
+    ## @param database.connectionInfo.port PostgreSQL port
+    ##
+    port: "5432"
+    ## @param database.connectionInfo.dbName PostgreSQL database name
+    ##
+    dbName: <database-name> # <ATTENTION> - Enter the PostgreSQL database name for testmonitor service
+    ## @param database.connectionInfo.user PostgreSQL username used by the service
+    ##
+    user: <username> # <ATTENTION> - Enter the PostgreSQL username for the testmonitor service
+    ## @param database.connectionInfo.secretName The name of an existing secret with
+    ## PostgreSQL connection credentials
+    ##
+    secretName: <secret-name> # <ATTENTION> - Enter the secret name for the testmonitor service with PostgreSQL connection credentials
+    ## @param database.connectionInfo.passwordKey Password key for database.connectionInfo.user to be retrieved from existing secret
+    ## NOTE: Ignored unless `database.connectionInfo.secretName` parameter is set.
+    ##
+    # passwordKey: "passkey"
+    ## @param database.connectionInfo.migrationPasswordKey Password key for database.connectionInfo.migrationUser to be retrieved from existing secret
+    ## If unset database.connectionInfo.passwordKey is used instead.
+    ## NOTE: Ignored unless `database.connectionInfo.secretName` parameter is set.
+    ##
+    # migrationPasswordKey: null
+
+  serviceAccount:
+    ## @param serviceAccount.annotations Annotations to add to the service account
+    ##
+    annotations:
+      {
+        iam.gke.io/gcp-service-account=<YOUR-GSA-NAME>@<YOUR-GOOGLE-CLOUD-PROJECT>.iam.gserviceaccount.com, # <ATTENTION> - Enter the CloudSQL service account name and the Google Cloud Project name
+      }
+    ## @param serviceAccount.name The name of the service account to use.
+    ## If not set and create is true, a name is generated using the fullname template
+    ##
+    name: ""
+
+dataframeservice:
```

Comment:
In dataframe service dremio also uses GCS, so it would be better if we add that configuration here.

Reply:
The GCS for DFS has been removed. Refer PR description for more info.